package.json

```diff
@@ -194,9 +194,7 @@
     "webpack-cli": "^6.0.1",
     "yargs": "^17.7.2"
   },
-  "resolutions": {
-    "react-is": "^19.0.0"
-  },
+  "resolutions": {},
```

Comment:
`useReactVersion` alters the `resolutions` entry.
If it's not present, the script fails.
I don't see a big difference in keeping an empty entry here and altering the script to create an empty entry if it does not exist. 🤔